static/src/javascripts/bootstraps/media.js

```diff
@@ -214,8 +214,13 @@ define([
                 // preroll for videos only
                 if (mediaType === 'video') {
 
+                    if (showEndSlate && detect.isBreakpoint({ min: 'desktop' })) {
+                        initEndSlate(player, endSlateUri);
+                    }
+
                     player.fullscreener();
-                    // Init plugins
+
+                    // do ads last so if the ad blocker throws an exception it doesn't stop anything else
```

Comment:
Rather than delaying execution, why not wrap in a try/catch block. Or do both. Whatever happens you should be catching the exception.